app-agent/src/bin/local.rs

```diff
@@ -19,7 +19,7 @@ fn main() {
         plugin_perf::PerfPlugin,
         plugin_procfs::ProcfsPlugin,
         plugin_csv::CsvPlugin,
-        plugin_socket_control::SocketControlPlugin,
+        //plugin_socket_control::SocketControlPlugin,
```

Comment:
```suggestion
        plugin_socket_control::SocketControlPlugin,
```

Reply:
I do not understand the purpose of this plugin ? for testing socket communication in Alumet pipeline ?, ok for add it

Reply:
@TheElectronWill